src/apps/store/models/order.ts

```diff
@@ -0,0 +1,29 @@
+import { prop, getModelForClass, modelOptions } from '@typegoose/typegoose';
+import { BaseModel } from '@shared/data/mongodb/base.model';
+
+class OrderItem {
+  @prop({ required: true })
+  public bookId: string;
+}
+
+@modelOptions({ schemaOptions: { collection: 'orders' } })
+export class Order extends BaseModel<Order> {
+  @prop({ required: true })
+  public userId: string;
+
+  @prop({ required: true, type: OrderItem })
+  public items: OrderItem[];
```

Comment:
Basically, the order item must contain the item information at the time that the customer creates the order. In this case, it should contain at least some information, such as the book's ID, title, and price.